code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -300,6 +300,7 @@ GLOBAL_LIST_INIT(sinew_recipes, list ( \
 	singular_name = "carp scales"
 	icon_state = "sheet-carp"
 	inhand_icon_state = "sheet-carp"
+	singular_name = "carp scale"
```

Comment:
I uh

Reply:
I uh i uhm uh

Reply:
No uh, the singular name of carp scales should not be "carp scales", the change was right, you just forgot to remove the old var

Reply:
+1